packages/orbit-components/src/Box/Box.stories.js

```diff
@@ -0,0 +1,272 @@
+// @flow
+import * as React from "react";
+import { storiesOf } from "@storybook/react";
+import { text, select, object } from "@storybook/addon-knobs";
+
+import Text from "../Text";
+import RenderInRtl from "../utils/rtl/RenderInRtl";
+import {
+  DISPLAY,
+  WRAP,
+  SHRINK,
+  GROW,
+  OVERFLOW,
+  COLORS,
+  DIRECTION,
+  ALIGN,
+  SPACINGS,
+  TEXT_ALIGN,
+  JUSTIFY,
+  WIDTH_AND_HEIGHT,
+  ELEVATION,
+  BORDER_RADIUS,
+  POSITION,
+} from "./consts";
+
+import Box from "./index";
+
+const DEFAULT_COLOR = "blueDark";
+
+storiesOf("Box", module)
+  .add(
+    "Default",
+    () => {
+      const children = text("Children", "Default box");
+      const as = text("As", "span");
+      return (
+        <Box as={as}>
+          <Text>{children}</Text>
+        </Box>
+      );
+    },
+    {
+      info:
+        "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+    },
+  )
+  .add("Overflow", () => {
+    const overflow = select("overflow", OVERFLOW, OVERFLOW.AUTO);
+
+    return (
+      <Box
+        background={DEFAULT_COLOR}
+        overflow={overflow}
+        color="cloudLight"
+        padding="large"
+        maxWidth="300px"
+      >
+        <Text type="white">Text</Text>
+      </Box>
+    );
+  })
+  .add("TextAlign", () => {
+    const textAlign = select("textAlign", TEXT_ALIGN, TEXT_ALIGN.CENTER);
+
+    return (
+      <Box
+        background={DEFAULT_COLOR}
+        textAlign={textAlign}
```

Comment:
The `align` is not visible since `Text` component has its own `text-align`. I would not use the `Text` inside and that's it.

Reply:
I wanted `Text` to have the correct `font-family`. But considering that this component can set `text-align`, maybe it makes sense that it sets `font-family` just like `Text`?

Reply:
That sounds great 🙂  Why not – it won't be any side effect that we would not like.

Reply:
There are still `Text` components the stories and `Box` should have `font-family` set so we don't have to use `Text` to avoid Times New Roman.